src/avt/Plotter/vtk/vtkParallelImageSpaceRedistributor.C

```diff
@@ -391,8 +398,19 @@ vtkParallelImageSpaceRedistributor::RequestData(
         {
             outgoingPolyData[i] = vtkPolyData::New();
             outgoingPolyData[i]->GetCellData()->CopyAllocate(inCD,outgoingCellCount[i]);
+#if LIB_VERSION_LE(VTK,8,1,0)
             int connSize = outgoingPointCount[i]+outgoingCellCount[i]; // overhead of one for each cell
+#else
```

Comment:
should this be removed now that we have removed VTK8 support? @brugger1 @biagas

Reply:
VTK8 support only removed from develop, not 3.4RC.

Reply:
Ah, that makes sense. Thanks!